docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -0,0 +1,171 @@
+- Name included in welcome announcement in email newsletters and social media
+- **New** 5 minute sponsored lightning talk on main stage of the conference
+- **New** One newsletter sponsorship (logo & 300 words) to our newsletter ({{ newsletter_subs }}+ subscribers)
+- **New** A logo on all Write the Docs website pages until the end of 2020. (30,000 pageviews/mo)
+- **New** A :doc:`small ad </sponsorship/website>` on the Write the Docs website (240x180px, 180 characters, 10,000 pageviews/mo) for 3 months
+
+The **Patron** package costs **{{sponsorship.patron.price}}**.
+
+Keystone
+--------
+
+:strike:`Limit 1` - **Sold Out**
+
+
+The **Keystone** package highlights you as our main community partner:
+
+- **Increased** Twenty (20) tickets_
+- **Increased** Ten (10) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A large description and logo (750 words) of your company on the conference website
+- A featured **virtual sponsorship booth**
+- A featured table at our **virtual job fair**
+- Large logo included in intermission slides and on talk videos
+- Name included in welcome announcement in email newsletters and social media
+- **New** 5 minute sponsored lightning talk on main stage of the conference
+- **New** Two newsletter sponsorships (logo & 300 words) to our newsletter ({{ newsletter_subs }}+ subscribers)
+- **New** A logo on all Write the Docs website pages until the end of 2020. (30,000 pageviews/mo)
+- **New** A :doc:`small ad </sponsorship/website>` on the Write the Docs website (240x180px, 180 characters, 10,000 pageviews/mo) for 3 months
+
+The **Keystone** package costs **{{sponsorship.keystone.price}}**.
+
+.. warning:: This is a draft prospectus and is subject to change.
+
+Other Sponsorship Opportunities
+===============================
+
+The following a la carte offerings are available either independently or
+combined with one of the previous packages to increase visibility at the event.
+
+Opportunity Grants
+------------------
+
+Provide additional money for our Opportunity Grant program,
+which provides funding for people to attend the conference.
+
+**{{sponsorship.second_draft.price}}**
+
+Additionally, for **{{sponsorship.publisher.price}}** we will offer exclusive sponsorship of our Opportunity Grant program.
+Please talk to us about this soon,
+as this option is removed once we have secured any sponsors for this benefit.
+
+Benefits
+~~~~~~~~
+
+* Your sponsor logo will be shown on the stage during all staff presentations as a grant sponsor (opening, closing). 
+* We will mention your company as a grant sponsor on Twitter from the official Write the Docs account
+
+Writing Day
+-----------
+
+Sponsor the Writing Day on Sunday, where we get together to help improve the documentation of many projects.
+This is great for any company that is looking for contributors to their open source projects.
```

Comment:
Should we make it explicit that companies are only allowed to source contributors for open-source projects (as opposed to internal/proprietary projects)?

Reply:
I don't think we make that explicit, really. People are often happy to work on company projects if it interests them, so I'm fine giving folks agency.